### DISCUSSION TITLE
Make Obsidian Leaflet available in Quartz!

### DISCUSSION THREAD

**Discussion by eddieilish** [javalent/obsidian-leaflet#429] [Ideas]
First things first, thank you for developing Obsidian Leaflet! Now, I rely on it for creating a "database" for a DnD campaign I'm playing with friends. It just recently happened that they asked me to share the info with them and I stumbled upon Quartz as an alternative to Obsidian Publish. Apparently, Quartz support some plugins but sadly Leaflet is not among them. I don't know is this is too niche but I think there's a growing amount of users that are uploading their vaults and that could benefit from this. In any case, thank you for reading!

**Comment by SpocWeb** [comment]
Hi @eddieilish I asked the same question and this is estimated a big effort: https://github.com/javalent/obsidian-leaflet/discussions/395

**Reply by eddieilish** [reply]
Oh! Good to know there's more people interested and eventrying to get it to work! Sadly I lack coding skills so I didn't even try. It's also sad to hear it takes such a huge effort, as it may not be worth coding. But if you manage, let me know!
